packages/editor/src/client/api/index.ts

```diff
@@ -6118,6 +6147,42 @@ export function usePeerProfileLazyQuery(baseOptions?: Apollo.LazyQueryHookOption
 export type PeerProfileQueryHookResult = ReturnType<typeof usePeerProfileQuery>;
 export type PeerProfileLazyQueryHookResult = ReturnType<typeof usePeerProfileLazyQuery>;
 export type PeerProfileQueryResult = Apollo.QueryResult<PeerProfileQuery, PeerProfileQueryVariables>;
+export const RemotePeerPrfoileDocument = gql`
+    query RemotePeerPrfoile($hostURL: String!, $token: String!) {
+  remotePeerProfile(hostURL: $hostURL, token: $token) {
+    ...FullPeerProfile
+  }
+}
+    ${FullPeerProfileFragmentDoc}`;
+
+/**
+ * __useRemotePeerPrfoileQuery__
+ *
+ * To run a query within a React component, call `useRemotePeerPrfoileQuery` and pass it any options that fit your needs.
+ * When your component renders, `useRemotePeerPrfoileQuery` returns an object from Apollo Client that contains loading, error, and data properties
+ * you can use to render your UI.
+ *
+ * @param baseOptions options that will be passed into the query, supported options are listed on: https://www.apollographql.com/docs/react/api/react-hooks/#options;
+ *
+ * @example
+ * const { data, loading, error } = useRemotePeerPrfoileQuery({
+ *   variables: {
+ *      hostURL: // value for 'hostURL'
+ *      token: // value for 'token'
+ *   },
+ * });
+ */
+export function useRemotePeerPrfoileQuery(baseOptions: Apollo.QueryHookOptions<RemotePeerPrfoileQuery, RemotePeerPrfoileQueryVariables>) {
+        const options = {...defaultOptions, ...baseOptions}
+        return Apollo.useQuery<RemotePeerPrfoileQuery, RemotePeerPrfoileQueryVariables>(RemotePeerPrfoileDocument, options);
+      }
+export function useRemotePeerPrfoileLazyQuery(baseOptions?: Apollo.LazyQueryHookOptions<RemotePeerPrfoileQuery, RemotePeerPrfoileQueryVariables>) {
```

Comment:
What does this lazy query does ?

Reply:
graphql-codegen creates two kind of queries. One to execute directly and another one that can be executed on a click. https://www.apollographql.com/docs/react/data/queries/#manual-execution-with-uselazyquery